site/cds_rdm/components.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# CDS-RDM is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS service component for subjects validation."""
+
+from invenio_drafts_resources.services.records.components import ServiceComponent
+from marshmallow import ValidationError
+
+
+class SubjectsValidationComponent(ServiceComponent):
+    """Service component for subjects validation."""
```

Comment:
Can we add a more descriptive comment? In this way, we will remember why we have this and we are doing this. Ping @kpsherva